app/javascript/dashboard/routes/dashboard/conversation/ContactPanel.vue

```diff
@@ -53,6 +53,16 @@
         </multiselect>
       </div>
     </div>
+    <contact-custom-attributes
+      v-if="hasContactAttributes"
+      :custom-attributes="contact.custom_attributes"
+    />
+    <conversation-labels :conversation-id="conversationId" />
+    <contact-conversations
```

Comment:
This can grow to a long list, please keep it at the end.

Reply:
I asked @iamsivin to move this up because 
1. I think tag is more of a ticket property which is often looked/scanned unlike the details like Timezone/browser etc.
2. The agent might add Labels on the go. So it should stay with Agent/Team/Labels which makes more sense because of similarity and ease off access. Also this won't grow more than 3 lines usually. 

Let me know if this makes sense.

Reply:
This makes sense. I agree with this. 

My point was around contact-conversations, not labels. The number of conversations can grow. I believe the metadata should be shown above conversations.